packages/shared/src/buttons/Button.js

```diff
@@ -8,9 +8,10 @@ import StyleSheet from '../PlatformStyleSheet';
 import type { StylePropType } from '../../index';
 
 type Props = {|
-  children: React.Node,
-  onPress?: () => void,
-  style?: StylePropType,
+  +children: React.Node,
+  +onPress?: () => void,
+  +style?: StylePropType,
+  +disabled?: boolean,
```

Comment:
We should refactor this (not now). What about making `onPress` mandatory instead?

Reply:
Definitely, It does not make sense to have a button with onPress

Reply:
> It does not make sense to have a button with onPress

Please elaborate... 😄

Reply:
I really should stop drinking while working 🍺 🤦‍♂️ 

It should have been without onPress